src/fonts/barlow-condensed.css

```diff
@@ -135,7 +135,7 @@
   font-family: 'Barlow Condensed';
   font-style: normal;
   font-display: auto;
-  font-weight: 600;
```

Comment:
Do not amend font-face rules. This has nothing to do with your desired changes.

Reply:
reverted.